docker-compose-watcher-mac.yml

```diff
@@ -15,24 +15,21 @@ services:
       timeout: 3s
       retries: 5
 
-  elixir-omg:
-    build: .
-    image: elixir-omg:dockercompose
-
   watcher:
-    image: elixir-omg:dockercompose
-    entrypoint: /bin/bash -c "./launcher.py && elixir --erl '-sname watcher' -S mix xomg.watcher.start --convenience --config ~/config_watcher.exs "
+    image: omisego/watcher:latest
+    command: "full_local"
+    build:
+      context: ./
+      dockerfile: Dockerfile.watcher
     environment:
-      - MIX_ENV=dev
-      - ELIXIR_SERVICE=WATCHER
       - ETHEREUM_RPC_URL=https://rinkeby.infura.io/v3/<your_api_key>
       - CHILD_CHAIN_URL=http://samrong.omg.network
       - ETHEREUM_NETWORK=RINKEBY
       - RINKEBY_CONTRACT_ADDRESS=0x740ecec4c0ee99c285945de8b44e9f5bfb71eea7
       - RINKEBY_TXHASH_CONTRACT=0x29f8cd44b4b94a148f779105f0e09e06f762b411ebef6c499281b74d45818c1c
       - RINKEBY_AUTHORITY_ADDRESS=0x41863dafbdf8cfc2a33fc38c0b525b6343d857b3
       - DATABASE_URL=postgres://omisego_dev:omisego_dev@docker.for.mac.localhost:5433/omisego_dev
-    restart: always
```

Comment:
Why was this here? @jbunce

Reply:
This needs to be here. It means the Docker container will be restarted if the BEAM dies.

Reply:
Which is odd, because the beam never dies. :-)
It either never starts, which means it just burns useless cycles that are related to wrong configuration. Or it starts and it works.

Reply:
I remember a time where it did ;)